src/css/base.scss

```diff
@@ -1735,6 +1735,10 @@ button.anchor {
     border-bottom: solid 1px var(--N100);
 }
 
+.dc__border-bottom-n2 {
+    border-bottom: solid 1px var(--N200);
```

Comment:
Please use existing `border-bottom` class.

Reply:
that is getting overwritten by border-bottom from nodeType.scss